fdbserver/ptxn/ProxyTLogPushMessageSerializer.h

```diff
@@ -26,70 +26,40 @@
 #include <cstdint>
 #include <unordered_map>
 
-#include "fdbclient/CommitTransaction.h"
+#include "fdbserver/ptxn/MessageTypes.h"
 #include "fdbserver/ptxn/Serializer.h"
 #include "flow/Error.h"
 
 namespace ptxn {
 
-const uint8_t ProxyTLogMessageProtocolVersion = 1;
+const SerializationProtocolVersion ProxyTLogMessageProtocolVersion = 1;
```

Comment:
It's better to give a meaningful name indicating this is related to ptxn messages, e.g.,
```suggestion
const SerializationProtocolVersion PTXN_TLOG_MSG_VERSION = 1;
```

Reply:
I would feel that the namespace should be sufficient to imply the purpose of the variable?